### DISCUSSION TITLE
Only block `ioctl` calls with specific arguments

### DISCUSSION THREAD

**Discussion by open-contracts** [netblue30/firejail#4581] [Q&A] [answered]
Hi, I would like to block the `ioctl` syscall to a specific driver, i.e. with a specific argument, only. Is this possible with firejail?

**Comment by rusty-snake** [comment]
ATM this is not possible.

**Comment by open-contracts** [comment, marked as answer]
I thought of a way, let me know if it is secure: 

simply block access to the device driver file `--blacklist=/dev/drivername` that would be the first argument to ioctl
